xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd

```diff
@@ -567,15 +584,29 @@ Rail transport, Roads and Road transport
 	</xsd:group>
 	<xsd:group name="SeatingValidityParametersGroup">
 		<xsd:annotation>
-			<xsd:documentation>Seating validity parameters for ACCESS RIGHT PARAMETER ASSIGNMENT.</xsd:documentation>
+			<xsd:documentation>Seating validity parameters for ACCESS RIGHT PARAMETER ASSIGNMENT. PassengerSeatRef is DEPRECATED -v2.0</xsd:documentation>
 		</xsd:annotation>
 		<xsd:sequence>
 			<xsd:element ref="TrainElementRef" minOccurs="0"/>
 			<xsd:element ref="TrainComponentLabelAssignmentRef" minOccurs="0"/>
+			<xsd:element ref="DeckPlanRef" minOccurs="0"/>
+			<xsd:element ref="DeckSpaceRef" minOccurs="0"/>
+			<xsd:group ref="SpotValidityParametersGroup"/>
 			<xsd:element ref="PassengerSeatRef" minOccurs="0"/>
 			<xsd:element ref="VehicleRef" minOccurs="0"/>
```

Comment:
While this element is now deprecated, my XML interpreter mentions that VehicleRef cannot be resolved anymore.

Reply:
It has the same problem here.  Fixed now